File: ruby1.8/nethunter.rb
require 'date'

startYear = 2012
endYear = startYear + 5
dt = Date.new(startYear, 1, 13)

while dt.year <= endYear
    if dt.wday == 5
        puts dt.strftime('%a, %d/%m/%Y')
    end
    dt = dt >> 1
end

File: python/nethunter.py
from dateutil import rrule
from datetime import date, datetime, timedelta

startYear = date.today().year
endYear = startYear + 5

for dt in rrule.rrule(rrule.MONTHLY, dtstart=datetime(startYear, 1, 13), until=datetime(endYear, 12, 13)):
	if dt.isoweekday() == 5:
		print dt.strftime('%a, %d/%m/%Y')
